support/src/indirect_intrusive_heap.h

```diff
@@ -304,7 +304,7 @@ namespace crimson {
     ConstIterator find(const I& ind_item) const {
       for (HeapIndex i = 0; i < count; ++i) {
 	if (data[i] == ind_item) {
-	  return ConstIterator(*this, i);
+	  return ConstIterator(this, i);
```

Comment:
And with ConstIterator invocations will change them to "*this".

Reply:
OK